plaso/parsers/wincc.py

```diff
@@ -0,0 +1,362 @@
+# -*- coding: utf-8 -*-
+"""Text parser plugin for WinCC log files."""
+
+from dfvfs.helpers import text_file
+
+from dfdatetime import time_elements as dfdatetime_time_elements
+
+from plaso.containers import events
+from plaso.lib import errors
+from plaso.parsers import interface
+from plaso.parsers import manager
+
+
+class SIMATICS7EventData(events.EventData):
+  """SIMATIC S7 event data.
+
+  Attributes:
+    creation_time (dfdatetime.DateTimeValues): date and time the log entry
+        was created.
+    message (str): the message of the event.
+  """
+
+  DATA_TYPE = 'wincc:simatic_s7:entry'
+
+  def __init__(self):
+    """Initializes event data."""
+    super(SIMATICS7EventData, self).__init__(data_type=self.DATA_TYPE)
+    self.creation_time = None
+    self.message = None
+
+
+class WinCCSysLogEventData(events.EventData):
+  """WinCC Sys Log event data.
+
+  Attributes:
+    log_identifier (int): identifier for this log file.
+    creation_time (dfdatetime.DateTimeValues): date and time the log entry
+        was created.
+    event_number (int): a number specifying the type of event.
+    unknown_int (int): an integer with unknown meaning.
+    unknown_str (str): a string with unknown meaning (mostly empty).
+    hostname (str): the hostname of the machine logging the event.
+    source (str): which device generated the event.
```

Comment:
message, source have a special purpose in Plaso (see: https://plaso.readthedocs.io/en/latest/sources/user/Output-and-formatting.html#dynamic-output-module-fields) use body (or equiv instead)

hostname can be used but will be can be set to a value determined by pre-processing https://github.com/log2timeline/plaso/blob/main/plaso/output/mediator.py#L127

remove unknown_int and unknown_str